README.md

```diff
@@ -72,6 +77,7 @@ You can also jump to a line with jk : *`<Leader>j{char}`* and  *`<Leader>k{char}
 |Html|Auto formatting|&#10004;|
 |css|Auto formatting|&#10004;|
 |json|Auto formatting|&#10004;|
+|C|Auto formatting|&#10004;|
```

Comment:
C is a real programming languague (not like css, html and json), so we should put goto, renaming, show doc status in README.